### PR TITLE
WIP: merge openmw master into Dreamweave

### PR DESCRIPTION
**Pull request by SaintMercury**

### CODE REVIEW

File: README.md
Comment (magicaldave) [inline README.md:1]: Prior to it becoming an issue, we should make sure to restore the tes3mp license information as opposed to the openmw one.

File: apps/openmw/mwmp/RecordHelper.cpp [hunk 1]
```diff
@@ -996,7 +996,7 @@ void RecordHelper::overrideRecord(const mwmp::MiscellaneousRecord& record)
             finalData.mData.mValue = recordData.mData.mValue;
 
         if (record.baseOverrides.hasKeyState)
-            finalData.mData.mIsKey = recordData.mData.mIsKey;
+            // finalData.mData.mIsKey = recordData.mData.mIsKey;
```
Comment (HotaruBlaze) [inline apps/openmw/mwmp/RecordHelper.cpp:999]: Is their a specific reason this is removed, or are we planning to add it back?

File: apps/openmw-mp/Script/Functions/RecordsDynamic.cpp [hunk 2]
```diff
@@ -1128,7 +1128,7 @@ void RecordsDynamicFunctions::SetRecordKeyState(bool keyState) noexcept
     unsigned short writeRecordsType = WorldstateFunctions::writeWorldstate.recordsType;
 
     if (writeRecordsType == mwmp::RECORD_TYPE::MISCELLANEOUS)
-        tempMiscellaneous.data.mData.mIsKey = keyState;
+        // tempMiscellaneous.data.mData.mIsKey = keyState;
```
Comment (HotaruBlaze) [inline apps/openmw-mp/Script/Functions/RecordsDynamic.cpp:1131]: This has also been commented out and probably needs to be fixed.

**Comment by SaintMercury** [comment]
I will likely close this PR and attempt a less WILD and full on merge alone in the future.

There was a LOT to keep track of in this PR and most of the changes are mostly unreviewable and were made in the moment to resolve conflicts.

**Pull request closed without merging**